tests/Unit/Helpers/Time/TimeSteppers/TimeStepperTestUtils.cpp

```diff
@@ -221,8 +220,8 @@ void integrate_error_test(const TimeStepper& stepper,
     // be dominated by the stepper.
     if (i > num_steps / 2) {
       REQUIRE(static_cast<bool>(y_error));
-      CHECK(abs(y - analytic(time.value())) < std::max(abs(*y_error), 1e-14));
-      CHECK(abs(y - analytic(time.value())) > abs(error_factor * *y_error));
+      CHECK(abs(y - analytic(time.value())) < std::max(abs(y_error), 1e-14));
+      CHECK(abs(y - analytic(time.value())) > abs(error_factor * y_error));
```

Comment:
Sorry, by local error do you mean that I should be comparing to the amount by which the error has changed over the last step? Or is there another measure that would be better?

Reply:
I think that would be better, yes.  The error that is really being estimated is the local truncation error for the single step being taken, which is somewhat hard to reason about because it depends on the history, which already has errors and probably doesn't even solve the ODE to the accuracy of the error estimate.

(The difference from the analytic solution is almost always going to grow exponentially in the long run, at which point this all becomes effectively meaningless, but we are assuming for this test that we are in a region where that behavior hasn't kicked in yet.)